Src/AutoFixture.xUnit.net.UnitTest/CompositeDataAttributeInsufficientDataTest.cs

```diff
@@ -74,17 +74,7 @@ public IEnumerator<object[]> GetEnumerator()
             yield return CreateTestCase(
                 data: new[]
                 {
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] {  new object[] { 1, 2    },  new object[] {  3,  4     }, new object[] { 5, 6 } }),

```

Comment:
It's not clear to me why this test case should be removed. :confused: Isn't it equivalent to the following test?

```csharp
public class Tests
{
    public static IEnumerable<object[]> Property1
    {
        get
        {
            yield return new[] { "1", "2" };
            yield return new[] { "3", "4" };
        }
    }

    public static IEnumerable<object[]> Property2
    {
        get
        {
            yield return new[] { "5", "6" };
        }
    }

    [Theory]
    [ComposeTwoProperties("Property1", "Property2")]
    public void DoSomething(object x, object y, object z)
    {
        Assert.NotNull(x);
        Assert.NotNull(y);
        Assert.NotNull(z);
    }
}

internal class ComposeTwoPropertiesAttribute : CompositeDataAttribute
{
    internal ComposeTwoPropertiesAttribute(
        string fst,
        string snd)
        : base(
            new PropertyDataAttribute(fst),
            new PropertyDataAttribute(snd))
    {
    }
}
```